UIMentoringSession/MenteesChallenges/Pets Main Card UI Chellenge/OleksandrKharchenko/AnimalsCells/Olkha_PetsModels.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+import UIKit
+
+struct OlKha_PetsViewModel {
+    let name: String
+    let age: String
+    let breadAnimal: String
+    let type: AnimalsTypes
+    let animal: UIImage
+}
+
+struct OlKha_FilterViewModel {
+    let title: String
+    let type: AnimalsTypes
+    let animal: UIImage
+}
+
+let itemsFilter: [OlKha_FilterViewModel] = [
```

Comment:
it is okay for the test UI challenge, but the wrong setup for real project. It is better to locate it in the controller, extension, or presenter for example